src/Input/Stepper.tsx

```diff
@@ -0,0 +1,257 @@
+      handleChange(0);
+    }
+  };
+
+  return (
+    <InputWrapper
+      description={description}
+      label={label}
+      size={size}
+      error={error}
+      info={info}
+    >
+      <div
+        {...otherProps}
+        css={{
+          maxWidth: stepperWidth,
+          paddingLeft: 8,
+          paddingRight: 8,
+          backgroundColor: disabled ? colors.silver.light : colors.white,
+          border: "1px solid",
+          borderColor: error
+            ? colors.red.base
+            : focused
+            ? colors.blue.base
+            : colors.silver.darker,
+          borderRadius: 4,
+          "&[data-force-focus-state]": {
+            borderColor: colors.blue.base,
+          },
+          ":hover,  &[data-force-hover-state]": {
+            borderColor:
+              !disabled && !error
+                ? colors.grey.light
+                : error
+                ? colors.red.base
+                : colors.silver.darker,
+          },
+        }}
+      >
+        <div
+          css={{ display: "flex", justifyContent: "center", height: "100%" }}
+        >
+          <button
+            type="button"
+            onClick={onDecrement}
+            disabled={atMin || disabled}
+            css={{
+              ...base.large,
+              paddingLeft: 4,
+              paddingRight: 4,
+              margin: 4,
+              display: "flex",
+              justifyContent: "center",
+              ":focus": {
+                outline: "none",
+              },
+              color: atMin || disabled ? colors.grey.lighter : colors.blue.base,
+              backgroundColor: "transparent",
+              border: "none",
+            }}
+          >
+            <IconMinus
+              css={{
+                width: 10,
+                height: 10,
+                color:
+                  disabled || atMin ? colors.grey.lighter : colors.blue.base,
+              }}
+            />
+          </button>
+
+          <input
+            css={{
+              ...base.large,
+              textAlign: "center",
+              border: "none",
+              // 34 + the 2px on either side for border = 36 (the height of the standard input)
+              height: 34,
+              width: 36,
+              ":focus": {
+                outline: "none",
+              },
+              boxSizing: "content-box",
+              backgroundColor: disabled ? colors.silver.light : colors.white,
+              color: disabled ? colors.grey.lighter : colors.black.base,
+            }}
+            disabled={disabled}
+            value={value}
+            onChange={onInputChange}
+            onFocus={handleFocus}
+            onBlur={handleBlur}
+          />
+
+          <button
+            type="button"
+            onClick={onIncrement}
+            disabled={atMax || disabled}
+            css={{
+              paddingLeft: 4,
+              paddingRight: 4,
+              margin: 4,
+              fontWeight: 300,
```

Comment:
setting `fontWeight` here around the SVG is a no-op yeah?

Reply:
whoops!